File: 1 - Sequencial/13.rb
#!/usr/bin/env ruby
#coding:utf-8
puts "Sexo: "
sexo = gets.chomp
puts "Altura: "
h = gets.chomp.to_f

if sexo.downcase == 'f'
  peso_ideal =  (62.1 * h) - 44.7
elsif sexo.downcase == 'm'
  peso_ideal = (72.7 * h) - 58
end

if defined? peso_ideal
  puts peso_ideal
end

File: 1 - Sequencial/13.py
#!/usr/bin/env python
#coding:utf-8
sexo = raw_input("Sexo: ")
h = float(input("Altura: "))

if sexo.lower() == 'f':
  peso_ideal =  (62.1 * h) - 44.7
elif sexo.lower() == 'm':
  peso_ideal = (72.7 * h) - 58

if 'peso_ideal' in locals():
  print peso_ideal